common/lib/xmodule/xmodule/course_module.py

```diff
@@ -740,6 +740,16 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    enable_unproctored_exams = Boolean(
+        display_name=_("Enable Unproctored Exams"),
+        help=_(
+            "Enter true or false. If this value is true, unproctored exams are enabled in your course. "
+            "If it is false, all exams will be required to be proctored."
```

Comment:
@andy-armstrong per our conversation, can you please also modify the Help text for "Enable Timed Exams"? (Line 765). "Enter true or false. If this value is true, timed exams are enabled in your course. Regardless of this setting, timed exams are enabled if **Enable Proctored Exams** is set to true.